temp-schema-generator/manifests.ts

```diff
@@ -128,6 +118,7 @@ export interface MetaEditorView {
 	editors: string[];
 	pathname: string;
 	weight: number;
+	label: string;
```

Comment:
There's a TODO somewhere to discuss if `label` should be required on all manifests so we know what to call them if we ever need to display anything regarding a specific manifest in the backoffice, e.g. a table of all manifests. What do you think about that?

Reply:
We currently have a `name` property on all manifests that is used for this. The label is used in cases where we need a shorter "display" version of the name. Ex. Trees, Editor Views etc.

Reply:
Yeah I know. Was just thinking of the TODO if we should remove it.

Reply:
I have added label to the section manifest which is the last one (for now) that needs it. I have also removed the TODO.